resources/assets/js/components/ApplicationReview/ReviewApplications.tsx

```diff
@@ -131,6 +136,16 @@ const ReviewApplications: React.StatelessComponent<ReviewApplicationsProps> = ({
           />
         </div>
       </div>
+      <div data-clone>
+        <div data-c-margin="bottom(1)">
+          <ActivityFeed
```

Comment:
Once ActivityFeed accepts a filterComments function, this should only allow comments with the four Applicant locationIds.